src/main/java/core/basesyntax/serviceimpl/StorageFillingImpl.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax.serviceimpl;
```

Comment:
if you do separate packages for interface and implementations - follow this structure 
```suggestion
package core.basesyntax.service.impl;
```